app/locale/en.coffee

```diff
@@ -622,6 +622,37 @@
     hear_about: "How did you hear about CodeCombat?"
     fill_fields: "Please fill out all fields."
     thanks: "Thanks! We'll send you setup instructions shortly."
+    
+  teachers_quote:
+    name: "Quote Form"
+    title: "Request a Quote"
+    subtitle: "Get CodeCombat in your classroom, club, school or district!"
+    phone_number: "Phone number"
+    phone_number_help: "Where can we reach you during the workday?"
+    role_label: "Your role"
+    role_help: "Select your primary role."
+    tech_coordinator: "Technology coordinator"
+    advisor: "Advisor"
+    principal: "Principal"
+    superintendent: "Superintendent"
+    parent: "Parent"
+    organization_label: "Name of School/District"
+    city: "City"
+    state: "State"
+    country: "Country"
+    num_students_help: "How many do you anticipate enrolling in CodeCombat?"
+    education_level_label: "Education Level"
+    education_level_help: "Choose as many as apply."
+    elementary_school: "Elementary School"
+    high_school: "High School"
+    please_explain: "(please explain)"
+    middle_school: "Middle School"
+    college_plus: "College or higher"
+    anything_else: "Anything else we should know?"
+    thanks_header: "Thanks for requesting a quote!"
+    thanks_p: "We'll be in touch soon. Questions? Email us:"
+    thanks_anon: "Login or sign up with your account below to access your two free enrollments (we’ll notify you by email when they have been approved, which usually takes less than 48 hours). As always, the first hour of content is free for an unlimited number of students."
+    thanks_logged_in: "Your two free enrollments are pending approval. We’ll notify you by email when they have been approved (usually within 48 hours). As always, the first hour of content is free for an unlimited number of students."
```

Comment:
I'm not aware of any decision or preference either way. I assumed it's easier for a translator to translate in this order, side by side with the page being translated.